From b8fc9bd07bc2ac222bf69719f32d9b2435990914 Mon Sep 17 00:00:00 2001
From: Harry Wentland <harry.wentland@amd.com>
Date: Sun, 24 Sep 2017 14:31:51 -0400
Subject: [PATCH 2404/4131] drm/amd/display: Remove unused
 dc_validate_guaranteed function

It got refactored away and was never cleaned.

Signed-off-by: Harry Wentland <harry.wentland@amd.com>
Acked-by: Alex Deucher <alexander.deucher@amd.com>
---
 drivers/gpu/drm/amd/display/dc/core/dc.c | 32 --------------------------------
 1 file changed, 32 deletions(-)

diff --git a/drivers/gpu/drm/amd/display/dc/core/dc.c b/drivers/gpu/drm/amd/display/dc/core/dc.c
index 655ad58..b0d013c 100644
--- a/drivers/gpu/drm/amd/display/dc/core/dc.c
+++ b/drivers/gpu/drm/amd/display/dc/core/dc.c
@@ -651,38 +651,6 @@ void dc_destroy(struct dc **dc)
 	*dc = NULL;
 }
 
-bool dc_validate_guaranteed(
-		struct dc *dc,
-		struct dc_stream_state *stream)
-{
-	enum dc_status result = DC_ERROR_UNEXPECTED;
-	struct dc_state *context;
-
-	if (!dc_validate_stream(dc, stream))
-		return false;
-
-	context = dc_create_state();
-	if (context == NULL)
-		goto context_alloc_fail;
-
-	dc_resource_state_construct(dc, dc->current_state);
-
-	result = dc->res_pool->funcs->validate_guaranteed(
-					dc, stream, context);
-
-	dc_release_state(context);
-
-context_alloc_fail:
-	if (result != DC_OK) {
-		dm_logger_write(dc->ctx->logger, LOG_WARNING,
-			"%s:guaranteed validation failed, dc_status:%d\n",
-			__func__,
-			result);
-		}
-
-	return (result == DC_OK);
-}
-
 static void program_timing_sync(
 		struct dc *dc,
 		struct dc_state *ctx)
-- 
2.7.4

